app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.java

```diff
@@ -802,16 +822,16 @@ public void onClick(View view) {
                 break;
 
             case R.id.add_to_homescreen:
-                final IWebView webView = getWebView();
-                if (webView == null) {
-                    return;
-                }
-
-                final Bitmap icon = webView.getIcon();
-                final String url = webView.getUrl();
-                final String title = webView.getTitle();
+                if (!isLoading()) {
```

Comment:
Why are we not showing this dialog if we are still loading? Shouldn't we hide or disable the menu item in this case too?